app/src/pages/inside/stepPage/modals/editDefectModals/makeDecisionModal/optionsStepForm/optionsSection.jsx

```diff
@@ -0,0 +1,209 @@
+        searchMode,
+      };
+      if (searchMode === SEARCH_MODES.WITH_FILTER) {
+        requestData.filterId = activeFilter.id;
+      }
+      setLoading(true);
+      let currentItemLogRequest = [];
+      let similarItemsRequest = [];
+      let requests;
+      if (isBulkOperation) {
+        currentTestItem.map((elem) => {
+          currentItemLogRequest.push(
+            fetch(URLS.logItemStackTrace(activeProject, elem.path, ERROR_LOGS_SIZE)),
+          );
+          return (
+            searchMode &&
+            similarItemsRequest.push(
+              fetch(URLS.logSearch(activeProject, elem.id), {
+                method: 'post',
+                data: requestData,
+              }),
+            )
+          );
+        });
+        requests = searchMode
+          ? currentItemLogRequest.concat(similarItemsRequest)
+          : currentItemLogRequest;
+      } else {
+        currentItemLogRequest = fetch(
+          URLS.logItemStackTrace(activeProject, currentTestItem.path, ERROR_LOGS_SIZE),
+        );
+        similarItemsRequest =
+          searchMode &&
+          fetch(URLS.logSearch(activeProject, currentTestItem.id), {
+            method: 'post',
+            data: requestData,
+          });
+        requests = searchMode
+          ? [currentItemLogRequest, similarItemsRequest]
+          : [currentItemLogRequest];
+      }
+
+      Promise.all(requests)
+        .then((responses) => {
+          const [currentItemRes, similarItemsRes] = responses;
+          const currentItemLogs = isBulkOperation
+            ? responses.map((item) => item.content)
+            : currentItemRes.content;
+          const items = [];
+          isBulkOperation
+            ? currentTestItem.map((elem, i) => {
+                return items.push({ ...elem, logs: currentItemLogs[i] });
+              })
+            : items.push({ ...currentTestItem, logs: currentItemLogs }, ...(similarItemsRes || []));
+          setModalState({
+            testItems: items,
+            selectedItems: isBulkOperation
+              ? items.map((item) => {
+                  return { ...item, itemId: item.id };
+                })
+              : items,
+          });
+          setLoading(false);
+        })
+        .catch(({ message }) => {
+          setModalState({
+            testItems: [],
+            selectedItems: [],
+          });
+          setLoading(false);
+          dispatch(
+            showNotification({
+              message,
+              type: NOTIFICATION_TYPES.ERROR,
+            }),
+          );
+        });
+    };
+
+    optionValue !== ALL_LOADED_TI_FROM_HISTORY_LINE && fetchLogs(modalState.searchMode);
+  }, [optionValue]);
+
+  return (
+    <div className={cx('options-section', { 'shown-less': rightSectionIsLess })}>
+      <ScrollWrapper hideTracksWhenNotNeeded autoHide>
+        <div className={cx('header-block')}>
+          <button className={cx('button')} onClick={() => setIsShown(!rightSectionIsLess)}>
```

Comment:
I suggest excluding collapsed button from the specific component to the common component if it's possible.